CONTRIBUTING.md

```diff
@@ -180,23 +180,37 @@ $ git push origin [name_of_your_new_branch]
 
 
 ### Setting up for cmd2 development
-For doing cmd2 development, you actually do NOT want to have cmd2 installed as a Python package.
-So if you have previously installed cmd2, make sure to uninstall it:
+For doing cmd2 development, it is recommended you create a virutal environment using Conda or Virtualenv and install the package from the source.
+
+#### Create a new environment for cmd2 using Conda
 ```sh
-$ pip uninstall cmd2
+$ conda create -n cmd2_py36 python=3.6
+$ source activate cmd2
```

Comment:
I would change this to `conda activate cmd2`. As of December 2017, the [Anaconda release notes](https://conda.io/docs/release-notes.html#new-feature-highlights) state:

> The logic and mechanisms underlying environment activation have been reworked. With conda 4.4, conda activate and conda deactivate are now the preferred commands for activating and deactivating environments. You’ll find they are much more snappy than the source activate and source deactivate commands from previous conda versions. The conda activate command also has advantages of (1) being universal across all OSes, shells, and platforms, and (2) not having path collisions with scripts from other packages like python virtualenv’s activate script.